### DISCUSSION TITLE
7.01 일일 회의

### DISCUSSION THREAD

**Discussion by jlee38266** [3sam5oh/openSearch-mini-project#10] [일일 회의]
### 해야 할 일 (오전 중)  
1.  terraform apply test
2. 기능 정리 (요구사항)
3. 현재 시도 중인 검색 엔진 메소드 갈무리 필요 (추후 어떤 방식을 채택할지 선택)

#### 김아영
- 이클립스 개발 툴 사용, 인코딩 이슈 csv (UTF-8로 변환 필요)  -> ES 활용으로 기능 구현 중

#### 이준형
- aws opensearch service 활용하여 검색 기능 구현 중, 현재 크롤링 도입, opensearch dashboard를 통한 조회 테스트 중 -> 검색 가능 (현재 기간상에서 로그 분석이나 모니터링 시도는 고민중),

![image](https://github.com/3sam5oh/openSearch-mini-project/assets/152199695/0eda1b3d-6412-44a1-84bf-f051a08d0ec2)

#### 이재원
- terraform 활용하여 CI/CD를 관리할 수 있는 기본 설계는 완료 (단, apply를 하지 않았기 때문에 apply 테스트가 필요). docker에서 opensearch api를 pull 받아서 시도 중, security 문제로 인해 dashboard를 로그인할 수 없는 이슈가 있습니다. 

#### 정태원
- ELK 스택 테스트중, docker image 받아서 docker-compose, 회사명이 완전히 일치해야만 조회가 되는 부분을 발견, ELK 사용 시 용량 제한 인지, postman을 활용하여 검색 쿼리를 통해 온전히 일치하는 경우에만 조회가 가능.

### 화면 구현
html, css, js를 활용하여 검색바와 박스형의 리스트를 보여주는 간단한 화면으로 구현

**Comment by t4e1** [comment]
1.opensearch - spring 디펜던시 및 코드 찾아보기
